src/main/scala/com/datastax/gatling/plugin/model/DseCqlAttributes.scala

```diff
@@ -22,33 +24,37 @@ import io.gatling.core.check.Check
   * @param statement        CQL Statement to be sent to Cluster
   * @param cl               Consistency Level to be used
   * @param cqlChecks        Data-level checks to be run after response is returned
-  * @param genericChecks    Low-level checks to be run after response is returned
-  * @param userOrRole       User or role to be used when proxy auth is enabled
-  * @param readTimeout      Read timeout to be used
   * @param idempotent       Set request to be idempotent i.e. whether it can be applied multiple times
-  * @param defaultTimestamp Set default timestamp on request, overriding current system time
+  * @param node             Set the node that should handle this query
```

Comment:
Isn't that `node` parameter new?  IIRC with the previous driver, it was not possible to directly choose which node would execute a query.  I am not sure how it should be used.  Does it replace the whitelist LBP ?

Reply:
I believe it's equivalent to setHost(Host) in the old API... just with a different type now